static/home.js

```diff
@@ -5,8 +5,7 @@
     document.getElementById("game-error").innerText = "";
   }
 
-  async function getRandomGame() {
-    clearRandomGameResponseElements();
+  function assignRandomGamesParameters() {
```

Comment:
This feels like it could be a separate data class.  Will need to see how that is done in Javascript